package.json

```diff
@@ -5,15 +5,10 @@
     "email": "msopentech@microsoft.com",
     "url": "https://github.com/AzureAD/azure-activedirectory-library-for-js"
   },
-  "licenses": [
-    {
-      "type": "Apache 2.0",
-      "url": "http://www.apache.org/licenses/LICENSE-2.0"
-    }
-  ],
+  "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/AzureAD/azure-activedirectory-library-for-js.git"
   },
   "version": "1.0.8",
```

Comment:
this version should also be updated if you're going to change the version to 1.0.9